guides/token-factory/backend/src/app/modules/token_factory/stores/counter.ts

```diff
@@ -0,0 +1,23 @@
+import { BaseStore } from 'lisk-sdk';
+
+export interface CounterStoreData {
+	counter: number;
+}
+
+export const counterKey = Buffer.alloc(0);
+
+export const counterStoreSchema = {
+	$id: '/token_factory/counter',
+	type: 'object',
+	required: ['counter'],
+	properties: {
+		counter: {
+			dataType: 'uint32',
```

Comment:
TokenID index uses uint64 while counter uses uint32 this would limit the amount of tokens you are able to create with the module.